migrations/v6-MIGRATION.md

```diff
@@ -0,0 +1,130 @@
+## v6 API changes
+
+### New
+
+| Name                   |
+|------------------------|
+| `PurchaseOption`       |
+| `GooglePurchaseOption` |
+| `GoogleStoreProduct`   |
+| `Price`                |
+| `PricingPhase`         |
+| `RecurrenceMode`       |
+
+### StoreProduct updates
+
+| Previous                      | New                                                                           |
+|-------------------------------|-------------------------------------------------------------------------------|
+| price                         | Moved to PricingPhase                                                         |
```

Comment:
If anyone has a better idea rather than "Moved to PricingPhase". We will probably have to write a more extended guide, so I figured this would be enough

Reply:
This will depend on whether it's a one time purchase or a subscription, since currently they live in different places...

Reply:
i think this is fine for now... we definitely will need to clean up before releasing, but this is just for our understanding so that we don't forget to call something out later